server/sonar-server/src/test/java/org/sonar/server/issue/IssueQueryServiceTest.java

```diff
@@ -62,35 +59,24 @@
 import static org.mockito.Matchers.anyCollection;
 import static org.mockito.Matchers.eq;
 import static org.mockito.Matchers.isA;
+import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
-@RunWith(MockitoJUnitRunner.class)
```

Comment:
Great ! Next step we should do on this service => Use DbTester / Ester (not related to this PR)